src/index.ts

```diff
@@ -0,0 +1,12 @@
+import http from 'http'
+import express from 'express'
+import userRouter from './userRouter'
+const app = express()
+
+const port: number = 5555
```

Comment:
much better if you save all your config in another file. Please use this library and put these values:

1. port
2. pathToDirectory
https://stackoverflow.com/questions/50813591/how-to-use-node-config-in-typescript